### DISCUSSION TITLE
ABACUS v3.1 to be released on Jan 1, 2023

### DISCUSSION THREAD

**Discussion by wenfei-li** [deepmodeling/abacus-develop#1437] [General]
ABACUS v3.1 is scheduled to be released on January 1, 2023. We plan to offer the following improvements:

## Features & Bug fixes
- Inclusion of validated implicit solvation model, dipole correction, and compensating charge correction, as well as tests and documents. The new features enable simulations in complex environments such as solid-liquid interfaces.
- Enable restart calculations with saved electronic wave functions, and provide a detailed account of how to restart a failed/unfinished calculation.
- Inclusion of LibRI, which will facilitate calculations with hybrid functional. Several bugs/constraints of the current implementation of hybrid functional will also be fixed, such as the erroneous output of the H(R) matrix and the discrepancy between results obtained from single-core and multi-core calculations.
- Implementation of new geometry relaxation algorithms which exhibit better convergence behavior for most cases. More options for constrained variable cell relaxation will be provided, such as fixing cell volume, fixing cell shape, and so on.
- Allowing information such as H(R), S(R), wave-functions, etc. to be written into separate files / appended to a single file for each MD step.

## Installation
- The linking of ELPA library will be made optional during installation.
- Installation with Conda will be made available.

## Documentation
- We are planning to host a FAQ session in ABACUS online documentation. Threads from Issues that are of general concern will be consolidated into Q&A format and kept in FAQ session for future reference.

**Comment by Satinelamp** [comment]


**Reply by wenfei-li** [reply]
we received your message

**Comment by likefallwind** [comment]
Cool!
